ui/src/components/view/SearchView.vue

```diff
@@ -211,6 +214,9 @@ export default {
         } else if (item === 'tags') {
           type = 'tag'
         }
+	if (item === 'projectwide') {
```

Comment:
@DK101010 Can you fix the indentation error

Reply:
```
/jenkins/workspace/acs-ui-builder/ui/src/components/view/SearchView.vue
  221:2  error  Mixed spaces and tabs  no-mixed-spaces-and-tabs
```

Reply:
@davidjumani sorry, github seems to be act strange with tabs and spaces. I've solved and checked with lint